client/state/data-layer/wpcom/test/plans-request.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useNock from 'test/helpers/use-nock';
+import { useSandbox } from 'test/helpers/use-sinon';
+import {
+	plansReceiveAction,
+} from 'state/plans/actions';
+
+import {
+	requestPlans,
+} from '../plans-request';
+
+import {
+	ACTION_PLANS_REQUEST_SUCCESS,
+	WPCOM_RESPONSE as wpcomResponse,
+} from 'state/plans/test/fixture';
+
+describe( 'wpcom-api', () => {
+	let sandbox, dispatch;
+
+	useSandbox( newSandbox => {
+		sandbox = newSandbox;
```

Comment:
Why do we need `sandbox` variable, if it's only used in the `useSandbox` callback?